lighthouse-extension/app/manifest.json

```diff
@@ -18,6 +18,7 @@
   },
   "permissions": [
     "activeTab",
+    "tabs",
```

Comment:
i remember you telling me, but remind me again why we now need this permission?

Reply:
let's definitely add this on demand as you suggested @wardpeet, grabbing a new permission might still disable the extension on update?

Reply:
I don't think we need it.  In https://stackoverflow.com/questions/48089670/detect-and-test-chrome-extension-using-puppeteer, I needed it to get the title of tab. We're not doing anything special with the tabs api here.

Reply:
The reason why I need the extension is the following function
```
 const queryOpts = {
        active: true,
        currentWindow: true,
      };

      chrome.tabs.query(queryOpts, (tabs => {
```
without it I couldn't get the url of the tab

Reply:
Yessssss. I repro'd this.
Basically I believe activeTab isn't enough because of however we're bypassing the popup and such. It's hitting some corner case where `activeTab` just isn't enough. +1 to dynamically adding tabs permission just for this case.